components/TheAsideNavResources.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <aside
+    class="opacity-transition block bg-gray-100 mt-8 -mx-4 lg:bg-transparent lg:mt-0 lg:mx-0 lg:inset-0 z-90 lg:mb-0 lg:static lg:h-auto lg:overflow-y-visible lg:pt-0 lg:w-1/4 lg:block"
+  >
+    <div
+      class="h-full overflow-y-auto scrolling-touch text-center lg:text-left lg:h-auto lg:block lg:relative lg:sticky lg:top-24"
+    >
+      <nav
+        class="pt-8 lg:overflow-y-auto lg:block lg:pl-0 lg:pr-8 sticky?lg:h-(screen-24)"
+      >
+        <h2
+          class="text-xl xl:text-2xl text-light-onSurfacePrimary dark:text-dark-onSurfacePrimary font-medium leading-normal mb-4 lg:pt-4"
+        >
+          Resourcs
```

Comment:
```suggestion
          Resources
```

Reply:
Too early review :P

Reply:
Sorry! Didn't mean it to be a review, just a passing comment....! 😬